samcli/commands/remote_invoke/cloud.py

```diff
@@ -0,0 +1,19 @@
+"""
+Command group for "cloud" suite for commands. It provides common CLI arguments, template parsing capabilities,
+setting up stdin/stdout etc
```

Comment:
I don't understand what do we mean by "template parsing capabilities"

Reply:
That's on me, I copied the text from `local.py` and missed updating the text there.